scripts/esbuild/build.ts

```diff
@@ -8,14 +10,32 @@ import { buildScreenshot } from './screenshot';
 import { buildSysNode } from './sys-node';
 import { buildTesting } from './testing';
 
-// the main entry point for the Esbuild-based build
-async function main() {
+// the main entry point for the build
+export async function run(rootDir: string, args: ReadonlyArray<string>) {
   const opts = getOptions(process.cwd(), {
-    isProd: !!process.argv.includes('--prod'),
-    isCI: !!process.argv.includes('--ci'),
-    isWatch: !!process.argv.includes('--watch'),
+    isProd: !!args.includes('--prod'),
+    isCI: !!args.includes('--ci'),
+    isWatch: !!args.includes('--watch'),
   });
 
+  try {
+    if (args.includes('--release')) {
+      await release(rootDir, process.argv);
```

Comment:
I think this needs to be `args` instead of `process.argv` to make sure we're sending the right subset of `process.argv` to this command:
```suggestion
      await release(rootDir, args);
```

Reply:
yup, will switch all those over